Utils/Dataflow/030_PDFAnalyzer/manager.py

```diff
@@ -67,8 +67,7 @@ def path_join(a, b):
 if __name__ == "__main__":
     try:
         import Tkinter
-        from tkFileDialog import askdirectory, askopenfilename,\
-            askopenfilenames, asksaveasfile
+        from tkFileDialog import askdirectory, askopenfilenames, asksaveasfile
```

Comment:
I still would add to the commit description the information about "which import was removed".
It would be obvious if it was something like
```
- import sys
```
, but it`s not the case. I had to calculate the answer... twice, already ;)